cmd/rhobs/requests.go

```
@@ -0,0 +1,714 @@
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/openshift/osdctl/pkg/k8s"
+	"github.com/openshift/osdctl/pkg/osdctlConfig"
+	"github.com/openshift/osdctl/pkg/utils"
```

golangci-lint (2.11.4)

[error] ST1019: package "github.com/openshift/osdctl/pkg/utils" is being imported more than once

(staticcheck)